src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs

```diff
@@ -26,6 +26,9 @@ public static string ToSignatureDisplayString(this ISymbol symbol)
             return symbol.ToDisplayString(SymbolDisplayFormats.SignatureFormat);
         }
 
+        public static bool IsExternallyVisible(this ISymbol symbol)
+            => symbol.GetResultantVisibility() == SymbolVisibility.Public;
```

Comment:
i don't like this method name.  it's *realllly* ambiguous as to what it means.  I'd much rather have it called HasPublicResultantVisibility, or just remove and have the callers check this.